libs/application/templates/passport-annulment/src/fields/PassportSelection/index.tsx

```diff
@@ -85,7 +85,7 @@ export const PassportSelection: FC<FieldBaseProps> = ({
   return (
     <Box>
       <RadioFormField
-        error={fieldErros}
+        error={fieldErrors}
```

Comment:
Hey, I can't comment on the lines below but could you look into whether the 
> `label: (application.externalData.nationalRegistry.data as any)`

can rather utilize the shared module type: `NationalRegistryIndividual`?

Reply:
@coderabbitai what do you think?